service/src/geopm/PlatformTopo.hpp

```diff
@@ -17,7 +17,8 @@
 extern "C"
 {
     /// @brief Identify host CPU.
-    int geopm_read_cpuid(void); // Only used in examples/geopm_platform_supported.cpp
+    int geopm_read_cpuid(void);
+    int geopm_topo_popen(const char *cmd, FILE **fid);
```

Comment:
Maybe it would be better to just re-declare declare this function in the test file that needs it.  Otherwise we are committing to the API for 2.0 since this is an installed header.  I hope we can get away from lscpu eventually.